packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -43,16 +50,24 @@ export function useForms(patientUuid: string, startDate?: Date, endDate?: Date,
   const encountersRes = useEncountersWithFormRef(patientUuid, startDate, endDate);
   const pastEncounters = encountersRes.data?.data?.results ?? [];
   const data = allFormsRes.data ? mapToFormCompletedInfo(allFormsRes.data, pastEncounters) : undefined;
+  const { htmlFormEntryForms } = useConfig() as ConfigObject;
```

Comment:
```suggestion
  const { htmlFormEntryForms } = useConfig<ConfigObject>();
```

Reply:
Oh... whoops, yeah, we'd need to update the version of core. I guess it's fine for now.